CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 ## Application
 
 We don't generally track changes in application here, but you could have a look at:
-- [Blog](http://ideas.trustroots.org/)
+- [Blog](https://ideas.trustroots.org/)
```

Comment:
I thought these comments were killed once I merged `master` into the `influxdb` branch. They disappeared from my list of comments, but apparently when I posted the review it added them all.